data/capture_file.go

```diff
@@ -240,49 +240,6 @@ func getFileTimestampName() string {
 	return time.Now().Format(time.RFC3339Nano)
 }
 
-// TODO DATA-246: Implement this in some more robust, programmatic way.
-func getDataType(methodName string) v1.DataType {
```

Comment:
moved here: https://github.com/viamrobotics/rdk/pull/4557/files#diff-67c0823aa24d3667ac603603cf934d7c0d988547e19a88cd2d48b3aa1337dc99R193

With the viaonCaptureAllFromCamera added to the binary data list.